inprocess/src/test/java/io/grpc/inprocess/InProcessTransportTest.java

```diff
@@ -87,7 +87,7 @@ protected ManagedClientTransport newClientTransport(InternalServer server) {
   }
 
   @Override
-  protected boolean sizesReported() {
+  protected boolean sizesReportedAtServer() {
```

Comment:
Isn't your changes in InProcessTest.java writing to serverStream.statsTraceCtx reporting the sizes at server now?

Reply:
Previously it was only at client side as the github bug talks about only client side memory leak. But in the latest commit, sizes are being reported on both server and client side.